executor/src/lib.rs

```diff
@@ -2,17 +2,19 @@ extern crate console_error_panic_hook;
 
 use futures::prelude::*;
 use jsonrpsee::core::client::Client;
+use std::collections::BTreeMap;
 use wasm_bindgen::prelude::*;
 
+mod proof;
 mod runner_api;
 mod task;
 use crate::runner_api::RpcApiClient;
-use smoldot::json_rpc::methods::HexString;
+use smoldot::json_rpc::methods::{HashHexString, HexString};
 
 #[wasm_bindgen]
 pub async fn get_metadata(code: &str) -> Result<JsValue, JsValue> {
     std::panic::set_hook(Box::new(console_error_panic_hook::hook));
-    _ = console_log::init_with_level(log::Level::Debug);
+    // _ = console_log::init_with_level(log::Level::Debug);
```

Comment:
do we still need those or not?

Reply:
we're not using it so I commented them in cause we use it later

Reply:
can we just leave it on?

Reply:
yes we can